base/monitoring/cadvisor/cadvisor.DaemonSet.yaml

```diff
@@ -26,7 +26,7 @@ spec:
       serviceAccountName: cadvisor
       containers:
         - name: cadvisor
-          image: index.docker.io/sourcegraph/cadvisor:4.5.1@sha256:9da386528adbdf755f38dab6a40f6c4dbbb489ea4c418b2412d1418b5b25d5ea
+          image: index.docker.io/sourcegraph/cadvisor:216430_2023-05-02_5.0-3cc9006de32c@sha256:6ea7f53807e4a559ee825ba2a0c4c3b3f721275f0b5ce0e979f4fdad8a4e478a
```

Comment:
afaik the only reason `main` is on 4.5.1 is that is what Beatrix last updated it to.
I used `sg ops update-images` because it made it easier to get everything on the same version with embeddings

The other deploy repos only go as fas as `2022-12-06` on `main`. No idea why. Something I need to find out.

As long as the release batch change process isn't affected I don't think the version on `main` makes much of a difference as deployments should use tagged versions